composer.json

```
@@ -9,7 +9,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": "^7.4 || ^8.0",
-        "silverstripe/framework": "4.13.x-dev",
+        "silverstripe/framework": "^4.10",
```

This used to be targeted at `^4.12`

```suggestion
        "silverstripe/framework": "^4.12",
```

---

Done